### DISCUSSION TITLE
cp: can't stat '/usr/data/helper-script/files/services/S58factoryreset': No such file or directory

### DISCUSSION THREAD

**Discussion by shapingstuff** [Guilouz/Creality-Helper-Script-Wiki#659] [Issues]
I've recently updated the script and get this error when running the helper script. I've tried deleting and re-cloning the repository but get the same error.

Any suggestions?

**Comment by joseviniciusnunes** [comment]
You probably cloned the project without changing the output folder for helper-script, rename it
